src/services/authentication/hooks/index.js

```diff
@@ -34,13 +64,11 @@ const isActivated = (account) => {
 exports.before = {
 	create: [
 		lowerCaseUsername,
-		auth.hooks.authenticate(['local', 'jwt', 'ldap', 'iserv', 'moodle']),
 		injectUserId,
+		(context) => { delete context.params.provider; return context; },
```

Comment:
Please extract this to a named function with documentation on why we need to do this